### DISCUSSION TITLE
Is the "target volatility" objective function currently available?

### DISCUSSION THREAD

**Discussion by ActurialCapital** [dcajasn/Riskfolio-Lib#134] [Q&A]
Hi,

Is the "target volatility" objective function currently available?
The [optimization](https://riskfolio-lib.readthedocs.io/en/latest/portfolio.html#Portfolio.Portfolio.optimization) method of the Portfolio class does not suggest it. But just double checking here...

Thanks,
J

**Comment by uhasan1** [comment]
I personally export the entire efficient frontier and then select at the desired volatility level after it is optimized for the selected preferred risk metric...hope that helps

**Reply by ActurialCapital** [reply]
Thanks @uhasan1 

It sounds like a heavy procedure, especially if you're dealing with loads of data. 

Do you have an example in hand?
